client/bower.json

```diff
@@ -2,28 +2,18 @@
   "name": "scrumble",
   "description": "an app to interact with Trello boards and write daily mails",
   "main": "gulpfile.js",
-  "authors": [
-    "nicgirault <nic.girault@gmail.com>"
-  ],
+  "authors": ["nicgirault <nic.girault@gmail.com>"],
   "license": "MIT",
-  "keywords": [
-    "scrum"
-  ],
+  "keywords": ["scrum"],
   "homepage": "https://github.com/theodo/scrumble",
   "moduleType": [],
-  "ignore": [
-    "**/.*",
-    "node_modules",
-    "bower_components",
-    "test",
-    "tests"
-  ],
+  "ignore": ["**/.*", "node_modules", "bower_components", "test", "tests"],
   "dependencies": {
     "angular-ui-router": "0.2.15",
-    "angular": "1.4.9",
+    "angular": "1.7.2",
```

Comment:
If you're updating dependencies I would strongly suggest to add package-lock.json or yarn.lock, otherwise the dependencies of the dependencies (and the non fixed dependencies) might become non compatible.

Reply:
I'd love to. Problem is, front dependencies are still managed by bower 😰

Reply:
Oops, didn't see that !